src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+
+You may recognize the effect from two popular operating systems: [Windows Vista](https://i.kinja-img.com/gawker-media/image/upload/s--9RLXARU4--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/trgz8yivyyqrpcnwscu5.png) and [iOS](https://static.businessinsider.com/image/51fd2822eab8eae16e00000b-750.jpg). Now [Chrome](https://www.chromestatus.com/feature/5679432723333120) can do it too.
+
+The following video demonstrates `backdrop-filter` and each individual filter:
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-kitchen_sink.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    <a href="https://codepen.io/robinrendle/pen/LmzLEL" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Technical Details
+{% Aside 'key-term' %}
+  backdrop refers to all painted content behind the element, except when the ancestor has filters, opacity or [a few other documented properties applied](https://drafts.fxtf.org/filter-effects-2/#BackdropRootTriggers).
+{% endAside %}
+
+CSS `backdrop-filter` applies one or more filters to the backdrop of an element, leaving the contents unchanged.
+
+In order for the filter to apply any augmentations, some portion of the overlaying element needs to be transparent. Alternatively, consider that opaque elements will not produce a backdrop-filter effect while fully transparent elements produce a full filtered backdrop.
+
+Backdrop-filter shares attributes with CSS filters in that they create a new [stacking context](https://www.w3.org/TR/CSS21/zindex.html) and/or [containing blocks](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) when the property is used properly, aka it's not 'none'. Note, only elements with absolute and fixed position decendants will get a containing block.
+
+### Summary
+1. Overlaying element OR it's background must be at least partially transparent
+1. Overlaying element will get a new stacking context
+1. Filters may be combined
+
+{% Aside 'warning' %}
+  The backdrop-filter feature is computationally intensive, as are filters in general, so it may impact the performance of sites using it.
+{% endAside %}
+
+
+## Using backdrop-filter
+Get started in one line of CSS.
+
+### Single Filter
+
+```css
+.blur-behind-me {
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
```

Comment:
I kind of feel like this header and the one after could use a bit of text. Otherwise the post feels a bit thin, like it's mostly a collection of codepens.

Reply:
what kid of more text? i could see more being there sure. 

the post i think it healthily mostly visual examples and links to code. this feature is interesting because of what it can do visually, figured that's what most people will be interested in. the code for it is minimal and not exciting, the results are. 🤷‍♂

Reply:
Also, are there not little bits of wisdom to share about these values. Examples include, problems they might run into, what they're useful for, what they're not good for.

Reply:
hm, struggling to find wisdom for under the headings. will continue to riff.